Fix CB projectfiles (#3626)
wesnoth/wesnoth: projectfiles/CodeBlocks/README.md

    NOTE: You must make sure to download the 32-bit version.
    Building a 64-bit Wesnoth executable on Windows is currently not supported and will
    fail with the SDK package provided in the next step.
	
	To resolve out of memory errors when creating debug builds, follow these steps:
	

> 	1. If you use 32-bit Windows, run Admin command prompt

Reviewer: ```suggestion
	1. If you use 32-bit Windows, run in admin command prompt
```

Reviewer 2: "run from an Administrator command prompt"

	```
	bcdedit/set IncreaseUserVa 3072
	```
	
	2. Install MASM32;
	

> 	3. Open `cmd`( as Admin too);

Reviewer: ```suggestion
	3. Open `cmd` (also as admin);
```

Reviewer 2: Will powershell do or must it be MS-DOS "cmd"?

Reviewer: @GregoryLundberg PowerShell would indeed work here, but I think it's best to keep the instructions simple and only talk about CMD.

	
	4. Run these commands in an admin command prompt 
	```
	cd C:\..\mingw32\libexec\gcc\i686-w64-mingw32\7.2.0
	```

> 	and

Reviewer: Please merge these code blocks together.

    after new dependencies are added to Wesnoth or its version requirements
    change.

> 	for versions > 1.12, follow these steps in `libraries.md` for updating libraries yourself.

Reviewer: ```suggestion
    for versions > 1.12, follow the steps in `libraries.md` to update libraries yourself.
```
